src/lambda/expression.rs

```diff
@@ -122,7 +123,13 @@ impl Expression {
                   .unwrap_or_default();
                 if ctx.req_ctx.server.get_enable_cache_control() && resp.status.is_success() {
                   if let Some(max_age) = max_age(&resp) {
-                    ctx.req_ctx.set_min_max_age(max_age.as_secs());
+                    ctx.req_ctx.set_min_max_age(max_age.as_secs() as i64);
+                  }
+                  if let Some(policy) = cache_policy(&resp) {
+                    ctx.req_ctx.set_cache_visibility(&policy.cachability);
+                    if Some(Cachability::NoCache) == policy.cachability {
+                      ctx.req_ctx.set_min_max_age(-1);
```

Comment:
why not use 0 here instead?

Reply:
async_graphql expects the max_age to be set to `-1` if there is `no-cache` directive to convert the `cache_control` attribute of `response` accordingly.